Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:12
```

Comment:
out of curiosity - what is this file for?

Reply:
It seems its a file that was in repo, when I last pulled, I made a stash then, so thats probably how it ended up here. @martinwenisch could you please confirm, if this file should be omitted? Thanks